test/built-ins/Object/prototype/isPrototypeOf/undefined-context-and-object-value-throws.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2011 the Sputnik authors. All rights reserved.
```

Comment:
```suggestion
// Copyright (C) 2019 Alexey Shvayka. All rights reserved.
```

Reply:
Please see 86ba713: this test isn't new, but rather renamed [`S15.2.4.6_A12.js`](https://github.com/tc39/test262/blob/f505a9c9c32c4d2335aa53c2347524cc935ecd75/test/built-ins/Object/prototype/isPrototypeOf/S15.2.4.6_A12.js), authored by Sputnik team. Same goes for [`null-this-and-object-arg-throws.js`](https://github.com/tc39/test262/blob/86ba7134d51c89850dd76bfef723f586a7f07dd7/test/built-ins/Object/prototype/isPrototypeOf/null-this-and-object-arg-throws.js).